js/ui/html_editor/modules/toolbar.js

```diff
@@ -472,11 +473,28 @@ if(Quill) {
             return $container;
         }
 
+        _checkItemOptions(item) {
+            const optionsInfo = [{
+                newName: 'name',
+                oldName: 'formatName'
+            }, {
+                newName: 'acceptedValues',
+                oldName: 'formatValues'
+            }];
+
+            each(optionsInfo, (index, optionName) => {
+                if(isObject(item) && isDefined(item[optionName.oldName])) {
```

Comment:
What about case when item old named option set to `null` or `undefined`? User will not get any error. Mb it's better to check if `oldName` exists in `item`?